### PR TITLE
update @redhat-cloud-services/frontend-components-config-utilities

### PR DESCRIPTION
**Pull request by redallen**
Tweak webpack config to work with the latest shared version of the webpack proxy.

Ci-beta is working:
image

...and ci-stable as well!
image

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -219,51 +220,45 @@ module.exports = (_env, argv) => {
...
+      port,
+      disableHostCheck: true,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept, Authorization',
+      },
+      https: useProxy,
+      ...proxy({
+        port,
+        env: `ci-${isBeta ? 'beta' : 'stable'}`,
```
Comment (dlabrecq) [inline webpack.config.js:240]: Thanks for the update, Zack. I'll test locally.

We'll probably want to use `process.env.CLOUDOT_ENV` here Vs `ci`, so we can run against prod.

Comment (redallen) [reply webpack.config.js:240]: Cool. As a reminder, this is just to control which branch of `insights-chrome-build` to checkout and serve locally on the page.

**Review by dlabrecq — APPROVED** [review]
I tested locally, but could only switch between ci-beta and ci-stable if we clear cookies -- despite restarting webpack. 

Created https://issues.redhat.com/browse/RHCLOUD-15212 to track that issue separately.

File: webpack.config.js (hunk 1 above)
Comment (dlabrecq) [reply webpack.config.js:240]: right. It's mostly to test nav changes with the local chrome

**Review by ddonahue007 — APPROVED** [review]
LGTM

**Pull request merged**